spras/containers.py

```diff
@@ -271,38 +357,54 @@ def run_container_singularity(container: str, command: List[str], volumes: List[
     else:
         singularity_options.extend(['--env', environment])
 
-    # Handle unpacking singularity image if needed. Potentially needed for running nested unprivileged containers
+    expanded_image = None
     if config.config.unpack_singularity:
-        # Split the string by "/"
         path_elements = container.split("/")
-
-        # Get the last element, which will indicate the base container name
```

Comment:
I do quite miss this comment, as `base_cont` wasn't descriptive. [It could be worthwhile to specify why the last element is the base container name]